pkg/network/openshift_sdn.go

```diff
@@ -54,12 +53,25 @@ func renderOpenShiftSDN(conf *operv1.NetworkSpec, manifestDir string) ([]*uns.Un
 	}
 	data.Data["NodeConfig"] = nodeCfg
 
+	kpcDefaults := map[string][]string{
+		"metrics-bind-address":    {"0.0.0.0"},
+		"metrics-port":            {"9101"},
+		"proxy-mode":              {"iptables"},
+		"iptables-masquerade-bit": {"0"},
```

Comment:
Is the default we've been using? The upstream default is 14.

Reply:
Yes. I'm not sure why, but we've always been overriding it. (Note that in addition to the defaults set in CNO, there's *another* set of defaults [applied by the openshift-sdn binary](https://github.com/openshift/origin/blob/v4.1.0/pkg/cmd/openshift-sdn/proxy.go#L45) at startup. The new code here combines the two sets of defaults so that after https://github.com/openshift/origin/pull/23156 when we pass `--proxy-config`, all of the configuration (except `HostnameOverride`) will be on the CNO side and openshift-sdn won't change any of it.

Reply:
Gotcha. Can't wait to get these all cleaned up.